tools/objc/BUILD

```
@@ -16,9 +16,15 @@ sh_binary(
     data = [
         ":make_hashed_objlist.py",
         ":xcrunwrapper",
+        ":libtool_check_unique",
     ],
 )
 
+filegroup(
```

```suggestion
cc_library(
```

I think we actually need the binary here?